index.html

```diff
@@ -64,8 +64,14 @@ <h1 class="heading-title">
             contact me.
           </p>
 
+          <audio controls>
+            <source src="images/audio.mp3" type="audio/mp3" />
+          </audio>
+
           <div class="social-links">
-            <h3 class="heading">LET'S CONNECT</h3>
+            <a href="#contact">
+              <h4 class="heading-connect">LET'S CONNECT</h4></a
```

Comment:
- [ ] Please change the `h4` tag to `h2` tag as the Heading levels should have a meaningful hierarchy, or you can change it to any other tag but not the heading tags.